logos/logos-orchestrator/tests/performance/stream_gap_probe.py

```
@@ -0,0 +1,88 @@
+async def main() -> None:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--url", default="http://localhost:8080/v1/chat/completions")
+    parser.add_argument("--logos-key", required=True)
+    parser.add_argument("--model", required=True)
+    parser.add_argument("--streams", type=int, default=3)
+    parser.add_argument(
+        "--threshold", type=float, default=0.5, help="Report gaps above this many seconds as stalls"
+    )
+    args = parser.parse_args()
+
+    async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.15.15)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)